test/entry.js

```diff
@@ -121,6 +122,35 @@ function removeTestData(done) {
   ], done);
 }
 
+function createCacheData(done) {
+  // these are needed by the below
+  app.configuration.setConfig('masterTenantID', testData.testTenant._id);
+  app.configuration.setConfig('masterTenantName', testData.testTenant.name);
+
+  var src = path.join(TEST_CACHE_DIR, Folders.Framework);
+
+  function _copyFramework() {
+    var dest = path.join(app.configuration.serverRoot, Folders.Temp, testData.testTenant._id);
+    async.series([
+      async.apply(fs.remove, dest),
+      async.apply(fs.ensureDir, dest),
+      async.apply(fs.symlink, src, path.join(dest, Folders.Framework), 'dir')
```

Comment:
this throws a EPERM Error on Windows:
Error: EPERM: operation not permitted, symlink 'D:\AT-Project\at_repo\adapt_authoring\test\.testcache\adapt_framework' -> 'D:\AT-Project\at_repo\adapt_authoring\temp\000000000000000000000000\adapt_framework'

Creating a junction fixes the issue on my machine:
async.apply(fs.symlink, src, path.join(dest, Folders.Framework), 'junction')

Reply:
Have changed this, let me know if it's working now :)